src/gui.cpp

```diff
@@ -0,0 +1,36 @@
+#include "gui.h"
+
+void GUI::clear_screen() {
+  // Mac/Linux
+#ifndef DEBUG
+  system("clear");
+#endif
```

Comment:
shouldn't this be 
```suggestion
#ifdef DEBUG
  system("clear");
#endif
```

Reply:
If the DEBUG flag is defined, calling clear_screen won't do anything. Otherwise, normal behavior will do a system("clear") (which makes it harder to debug behavior). I can leave a comment for clarity perhaps?

Reply:
a comment would be good

Reply:
left a comment in function doc